library/src/main/java/com/bumptech/glide/load/engine/cache/MemorySizeCalculator.java

```diff
@@ -109,13 +108,9 @@ private String toMb(int bytes) {
     return Formatter.formatFileSize(context, bytes);
   }
 
-  @TargetApi(Build.VERSION_CODES.KITKAT)
   private static boolean isLowMemoryDevice(ActivityManager activityManager) {
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
-      return activityManager.isLowRamDevice();
-    } else {
-      return Build.VERSION.SDK_INT < Build.VERSION_CODES.HONEYCOMB;
-    }
+    return Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT
```

Comment:
Oh yeah... I have some vague recollection. I think you're right, I'll change this.